resources/migrations/20200924062956-create-timers-and-time-spans.up.sql

```diff
@@ -0,0 +1,30 @@
+CREATE TABLE IF NOT EXISTS timers(
+    id UUID primary key,
```

Comment:
You can add a default value for the primary key

Reply:
Some thoughts:

- There seems to be opinion in the DB community that a DB should not generate and return values, only store them. This prevents the application getting complected with a particular DB implementation. I understand that `bigserial` is seen as an anti-pattern for systems that expect to store significant amounts of data (not really a problem for us, though).

- Also, there are many types / versions of UUIDs. I'm wondering who should enforce the constraint on the type of UUID? The application or the database? If the DB creates the constraint, how to make the application respect it? As it stands we have parity in generation and in checking, because `java.util.UUID/randomUUID` and PG's `gen_random_uuid()` both return v4 UUIDs, and a cursory reading suggests that `clojure.core/uuid?` would validate any PG-generated UUID.

Reply:
I'll use PG's default generator as discussed. This should not be a problem in our case. 

If, later, we find that it threatens to become problematic, we can migrate away from it and put UUID creation in the app logic.

Reply:
Done. PG now generates UUID for us.